inc/classes/CDN/RocketCDN/APIClient.php

```diff
@@ -78,7 +78,7 @@ private function get_remote_subscription_data() {
 		}
 
 		$data = json_decode( $data, true );
-		$data = array_intersect_key( $data, $default );
+		$data = array_intersect_key( (array) $data, $default );
```

Comment:
To make sure to not trigger a notice when doing `if ( 'running' === $subscription_data['subscription_status'] ) {`:
```suggestion
		$data = array_intersect_key( (array) $data, $default );
		$data = array_merge( $default, $data );
```